docs/sources/panels/visualizations/logs-panel.md

```diff
@@ -17,6 +17,10 @@ The logs panel shows the result of queries that were entered in the Query tab. T
 
 To limit the number of lines rendered, you can use the **Max data points** setting in the **Query options**. If it is not set, then the data source will usually enforce a default limit.
 
+## Log level
+
+For logs with a **level** label, we use the value of the label to determine log level and update color accordingly. If log doesn't have a level label, we parse the log line to find out if its content matches any of the supported expressions. Log level is always determined by the first match. In a case where Grafana is not able to determine log level, log will be visualised with **unknown** log level. See [supported log levels and mappings of log level abbreviation and expressions](../explore/#log-level/).
```

Comment:
```suggestion
For logs where a **level** label is specified, we use the value of the label to determine the log level and update color accordingly. If the log doesn't have a level label specified, we parse the log to find out if its content matches any of the supported expressions (see below for more information). The log level is always determined by the first match. In case Grafana is not able to determine a log level, it will be visualized with **unknown** log level. See [supported log levels and mappings of log level abbreviation and expressions](../explore/#log-level/).
```